virtool/settings/data.py

```diff
@@ -0,0 +1,41 @@
+
+from virtool_core.models.settings import Settings
+
+from virtool.mongo.core import DB
+from virtool.settings.db import get, update
+from virtool.settings.oas import UpdateSettingsSchema
+
+
+class SettingsData:
+    def __init__(self, db):
+        self._db: DB = db
+
+    async def get_all(self) -> Settings:
+        """
+        List all settings.
+
+        :return: settings object
+
+        """
+
+        settings = Settings(**await get(self._db))
```

Comment:
Merge the `get()` function into this method. It shouldn't be necessary anymore. The data layer should always be used to access settings.